src/Ng/Jobs/EndpointMonitoringJob.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NuGet.Services.Configuration;
+using NuGet.Services.Metadata.Catalog;
+using NuGet.Services.Metadata.Catalog.Monitoring;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol;
+using NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test;
+using System.Linq;
```

Comment:
is there a way to make VS do this automatically? it doesn't and it's nontrivial at this point to do it manually